foreman/foreman.spec

```diff
@@ -848,6 +852,7 @@ rm -rf %{buildroot}
 %{_datadir}/%{name}/bundler.d/jsonp.rb
 %exclude %{_datadir}/%{name}/bundler.d/openid.rb
 %exclude %{_datadir}/%{name}/bundler.d/test.rb
+%exclude %{_datadir}/%{name}/bundler.d/foreman_tasks.rb
```

Comment:
1. Keep this list alphabetical.
1. The file in the foreman SRPM seems more precise than the one from `rubygem(foreman-tasks)`, shouldn't it be preferred?